src/dbup-sqlserver/SqlTableJournal.cs

```diff
@@ -42,8 +54,9 @@ protected override string CreateSchemaTableSql(string quotedPrimaryKeyName)
 $@"create table {FqSchemaTableName} (
     [Id] int identity(1,1) not null constraint {quotedPrimaryKeyName} primary key,
     [ScriptName] nvarchar(255) not null,
-    [Applied] datetime not null
+    [Applied] datetime not null,
+    [Hash] varchar(100) null
```

Comment:
Hi @ajhall , do you think storing the hash as a varchar will cause any issues? I am wondering if we could have a problem where the hash isn't converted correctly and we accidentally ship a stored procedure.

Reply:
I think it should be OK. We're using a SHA-256 hasher, which always returns a 256-bit value usually represented as a 64-character alphanumeric string.

We could store it as a BINARY(64) for maximum efficiency, but for simplicity of debugging and keeping the hash column easily human-readable I like the idea of a VARCHAR better.

https://stackoverflow.com/questions/252156/sql-server-2005-what-data-type-to-use-to-store-passwords-hashed-by-sha-256-algo

Reply:
I agree with going forward with human-readable storage. Should we encode the hash in base64 instead of UTF-8 before storing in the database?

Reply:
Here's an example of what a current entry in the database looks like:
`40af3382b4d703ab0286d6d94dc82ee7a8fb531eab8ed9ecc152ceeaaa9eb2e4`

That's typically how you would represent SHA-256 in human-readable form, such as in the output of the `Get-FileHash` command, so that's how I'd like to keep it unless we have a strong reason to represent it in another form.

Reply:
My bad, BitConverter.ToString() implementation is encoding using hex. Since are not worried about performance between base64 and hex, I believe we can resolve this comment.